From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 4 Jul 2015 10:13:09 +0800
Subject: [PATCH] {For LLVMLinux} Disable the use of clang integrated assembler

---
 Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/Makefile b/Makefile
index 98cb2facc047..68629afbdb86 100644
--- a/Makefile
+++ b/Makefile
@@ -411,6 +411,12 @@ KBUILD_AFLAGS_MODULE  := -DMODULE
 KBUILD_CFLAGS_MODULE  := -DMODULE
 KBUILD_LDFLAGS_MODULE := -T $(srctree)/scripts/module-common.lds
 
+ifeq ($(cc-name),clang)
+# Use gas because IA does not support the gas assembly extensions used in the kernel code.
+KBUILD_CFLAGS += -no-integrated-as
+KBUILD_AFLAGS += -no-integrated-as
+endif
+
 # Read KERNELRELEASE from include/config/kernel.release (if it exists)
 KERNELRELEASE = $(shell cat include/config/kernel.release 2> /dev/null)
 KERNELVERSION = $(VERSION)$(if $(PATCHLEVEL),.$(PATCHLEVEL)$(if $(SUBLEVEL),.$(SUBLEVEL)))$(EXTRAVERSION)
@@ -707,6 +713,7 @@ KBUILD_CPPFLAGS += $(call cc-option,-Qunused-arguments,)
 #KBUILD_CFLAGS += $(call cc-disable-warning, unused-variable)
 #KBUILD_CFLAGS += $(call cc-disable-warning, format-invalid-specifier)
 #KBUILD_CFLAGS += $(call cc-disable-warning, gnu)
+
 # Quiet clang warning: comparison of unsigned expression < 0 is always false
 #KBUILD_CFLAGS += $(call cc-disable-warning, tautological-compare)
 # CLANG uses a _MergedGlobals as optimization, but this breaks modpost, as the
-- 
