From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 4 Sep 2015 15:55:59 +0200
Subject: [PATCH] {LLVMLinux} (NOT UPSTREAMABLE) disable lib/test_static_keys
 when compiling with clang

---
 lib/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/lib/Makefile b/lib/Makefile
index 45f078784d6d..7b846b3ba5dd 100644
--- a/lib/Makefile
+++ b/lib/Makefile
@@ -40,7 +40,10 @@ obj-$(CONFIG_TEST_KSTRTOX) += test-kstrtox.o
 obj-$(CONFIG_TEST_LKM) += test_module.o
 obj-$(CONFIG_TEST_RHASHTABLE) += test_rhashtable.o
 obj-$(CONFIG_TEST_USER_COPY) += test_user_copy.o
+# Disable static keys with clang, which does not support defining functions as it is done
+ifneq ($(cc-name),clang)
 obj-$(CONFIG_TEST_STATIC_KEYS) += test_static_keys.o
+endif
 obj-$(CONFIG_TEST_STATIC_KEYS) += test_static_key_base.o
 obj-$(CONFIG_TEST_PRINTF) += test_printf.o
 
-- 
