From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 6 Aug 2016 10:00:49 +0200
Subject: [PATCH] {TYPO} init/do_mounts: fix typo in comment

---
 block/early-lookup.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/early-lookup.c b/block/early-lookup.c
index 3effbd0d35e9..3bc6eb7a7fc4 100644
--- a/block/early-lookup.c
+++ b/block/early-lookup.c
@@ -236,7 +236,7 @@ static int __init devt_from_devnum(const char *name, dev_t *devt)
  *	9) PARTLABEL=<name> with name being the GPT partition label.
  *	   MSDOS partitions do not support labels!
  *
- *	If name doesn't have fall into the categories above, we return (0,0).
+ *	If name doesn't fall into the categories above, we return (0,0).
  *	block_class is used to check if something is a disk name. If the disk
  *	name contains slashes, the device name has them replaced with
  *	bangs.
-- 
