From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 6 Aug 2016 10:00:49 +0200
Subject: [PATCH] {TYPO} init/do_mounts: fix typo in comment

---
 init/do_mounts.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/do_mounts.c b/init/do_mounts.c
index 1aef61e4feb8..1a88e7a9248d 100644
--- a/init/do_mounts.c
+++ b/init/do_mounts.c
@@ -213,7 +213,7 @@ static int match_dev_by_label(struct device *dev, const void *data)
  *	9) PARTLABEL=<name> with name being the GPT partition label.
  *	   MSDOS partitions do not support labels!
  *
- *	If name doesn't have fall into the categories above, we return (0,0).
+ *	If name doesn't fall into the categories above, we return (0,0).
  *	block_class is used to check if something is a disk name. If the disk
  *	name contains slashes, the device name has them replaced with
  *	bangs.
-- 
