From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 15 Aug 2016 09:53:04 +0200
Subject: [PATCH] {TYPO} lib/Kconfig.ubsan: fix "uninitialized" spelling

---
 lib/Kconfig.ubsan | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/Kconfig.ubsan b/lib/Kconfig.ubsan
index 39494af9a84a..35ef99a84f59 100644
--- a/lib/Kconfig.ubsan
+++ b/lib/Kconfig.ubsan
@@ -14,8 +14,8 @@ config UBSAN_SANITIZE_ALL
 	depends on UBSAN
 	depends on ARCH_HAS_UBSAN_SANITIZE_ALL
 
-	# We build with -Wno-maybe-uninitilzed, but we still want to
-	# use -Wmaybe-uninitilized in allmodconfig builds.
+	# We build with -Wno-maybe-uninitialized, but we still want to
+	# use -Wmaybe-uninitialized in allmodconfig builds.
 	# So dependsy bellow used to disable this option in allmodconfig
 	depends on !COMPILE_TEST
 	default y
-- 
