From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Wed, 17 Jun 2015 15:33:54 +0800
Subject: [PATCH] {PRAGMA} arch/x86/xen/mmu: ignore an intentional (?) array
 overflow

---
 arch/x86/xen/mmu.c | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/x86/xen/mmu.c b/arch/x86/xen/mmu.c
index c913ca4f6958..432542cbdedb 100644
--- a/arch/x86/xen/mmu.c
+++ b/arch/x86/xen/mmu.c
@@ -1113,8 +1113,18 @@ static void __init xen_cleanhighmap(unsigned long vaddr,
 
 	/* NOTE: The loop is more greedy than the cleanup_highmap variant.
 	 * We include the PMD passed in on _both_ boundaries. */
+/* clang says: the pointer incremented by 4096 refers past the end of the
+ *array (that contains 512 elements) [-Werror,-Warray-bounds-pointer-arithmetic]
+ */
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds-pointer-arithmetic"
+#endif
 	for (; vaddr <= vaddr_end && (pmd < (level2_kernel_pgt + PAGE_SIZE));
 			pmd++, vaddr += PMD_SIZE) {
+#ifdef __clang__
+#pragma GCC diagnostic pop
+#endif
 		if (pmd_none(*pmd))
 			continue;
 		if (vaddr < (unsigned long) _text || vaddr > kernel_end)
-- 
