From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Tue, 1 Sep 2015 11:22:07 +0200
Subject: [PATCH] {PRAGMA} crypto: poly1305: ignore documented structure abuse

---
 arch/x86/crypto/poly1305_glue.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/x86/crypto/poly1305_glue.c b/arch/x86/crypto/poly1305_glue.c
index 4a1c05dce950..4ae8225a5c76 100644
--- a/arch/x86/crypto/poly1305_glue.c
+++ b/arch/x86/crypto/poly1305_glue.c
@@ -85,8 +85,16 @@ static unsigned int poly1305_simd_blocks(struct poly1305_desc_ctx *dctx,
 			}
 			memcpy(sctx->u + 5, sctx->u, sizeof(sctx->u));
 			poly1305_simd_mult(sctx->u + 5, dctx->r.r);
+/* See the comment in struct poly1305_simd_desc_ctx definition to understand why this is ok */
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds-pointer-arithmetic"
+#endif
 			memcpy(sctx->u + 10, sctx->u + 5, sizeof(sctx->u));
 			poly1305_simd_mult(sctx->u + 10, dctx->r.r);
+#ifdef __clang__
+#pragma GCC diagnostic pop
+#endif
 			sctx->wset = true;
 		}
 		blocks = srclen / (POLY1305_BLOCK_SIZE * 4);
-- 
