From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Thu, 16 Apr 2015 11:00:06 +0800
Subject: [PATCH] {PRAGMA} ignore broken clang -Wshift-count-overflow check for
 ternary

---
 include/linux/dma-mapping.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/include/linux/dma-mapping.h b/include/linux/dma-mapping.h
index 55cb6983d610..53d2d9410969 100644
--- a/include/linux/dma-mapping.h
+++ b/include/linux/dma-mapping.h
@@ -136,7 +136,15 @@ struct dma_map_ops {
 extern const struct dma_map_ops dma_direct_ops;
 extern const struct dma_map_ops dma_virt_ops;
 
+#ifdef __clang__
+#define DMA_BIT_MASK(n) \
+_Pragma("GCC diagnostic push") \
+_Pragma("GCC diagnostic ignored \"-Wshift-count-overflow\"") \
+    (((n) == 64) ? ~0ULL : ((1ULL<<(n))-1)) \
+_Pragma("GCC diagnostic pop")
+#else
 #define DMA_BIT_MASK(n)	(((n) == 64) ? ~0ULL : ((1ULL<<(n))-1))
+#endif
 
 #define DMA_MASK_NONE	0x0ULL
 
-- 
