From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Wed, 17 Jun 2015 16:34:10 +0800
Subject: [PATCH] {PRAGMA} net/wireless/mwl8k: ignore intentional array bounds
 violation

---
 drivers/net/wireless/marvell/mwl8k.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/net/wireless/marvell/mwl8k.c b/drivers/net/wireless/marvell/mwl8k.c
index 47fb4b3ea004..0b7e0018c2e4 100644
--- a/drivers/net/wireless/marvell/mwl8k.c
+++ b/drivers/net/wireless/marvell/mwl8k.c
@@ -821,7 +821,15 @@ static inline void mwl8k_remove_dma_header(struct sk_buff *skb, __le16 qos)
 	if (hdrlen != sizeof(tr->wh)) {
 		if (ieee80211_is_data_qos(tr->wh.frame_control)) {
 			memmove(tr->data - hdrlen, &tr->wh, hdrlen - 2);
+/* clang doesn't like to reference 2 bytes before the beginning of the array */
+#ifdef __clang__
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Warray-bounds-pointer-arithmetic"
+#endif
 			*((__le16 *)(tr->data - 2)) = qos;
+#ifdef __clang__
+#pragma GCC diagnostic pop
+#endif
 		} else {
 			memmove(tr->data - hdrlen, &tr->wh, hdrlen);
 		}
-- 
