From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 15 Jan 2016 13:51:48 +0100
Subject: [PATCH] {BUG} drivers/edac/sb_edac: uninitialized var

---
 drivers/edac/sb_edac.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/edac/sb_edac.c b/drivers/edac/sb_edac.c
index 6744d88bdea8..14e0f25ecfa6 100644
--- a/drivers/edac/sb_edac.c
+++ b/drivers/edac/sb_edac.c
@@ -849,6 +849,7 @@ static char *knl_show_interleave_mode(u32 reg)
 		break;
 	default:
 		WARN_ON(1);
+		s = ""; /* BUG: uninitialized s (-Wsometimes-uninitialized from clang) */
 		break;
 	}
 
-- 
