From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 20 Dec 2021 20:38:30 +0100
Subject: [PATCH] {BUG in 5.16-rc2} net/mptcp: remove unused-but-set parameter

---
 net/mptcp/options.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/mptcp/options.c b/net/mptcp/options.c
index 355f798d575a..1ddeb3c909e6 100644
--- a/net/mptcp/options.c
+++ b/net/mptcp/options.c
@@ -580,7 +580,7 @@ static bool mptcp_established_options_dss(struct sock *sk, struct sk_buff *skb,
 			opts->ext_copy = *mpext;
 		}
 
-		remaining -= map_size;
+		/*remaining -= map_size;*/
 		dss_size = map_size;
 		if (skb && snd_data_fin_enable)
 			mptcp_write_data_fin(subflow, skb, &opts->ext_copy);
-- 
