From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Wed, 24 Jul 2019 08:31:43 +0200
Subject: [PATCH] {BUG in 5.3-rc1} block/sed-opal: add missing =

---
 block/sed-opal.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/block/sed-opal.c b/block/sed-opal.c
index 7e1a444a25b2..a857658da8df 100644
--- a/block/sed-opal.c
+++ b/block/sed-opal.c
@@ -130,7 +130,7 @@ static const u8 opaluid[][OPAL_UID_LENGTH] = {
 
 	/* tables */
 
-	[OPAL_TABLE_TABLE]
+	[OPAL_TABLE_TABLE] =
 		{ 0x00, 0x00, 0x00, 0x01, 0x00, 0x00, 0x00, 0x01 },
 	[OPAL_LOCKINGRANGE_GLOBAL] =
 		{ 0x00, 0x00, 0x08, 0x02, 0x00, 0x00, 0x00, 0x01 },
-- 
