From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Wed, 25 May 2016 21:57:43 +0200
Subject: [PATCH] {BUG?} nconf: fix -Wmacro-redefined for _GNU_SOURCE

---
 scripts/kconfig/nconf.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/scripts/kconfig/nconf.c b/scripts/kconfig/nconf.c
index 2587085c5ee5..600da35e779e 100644
--- a/scripts/kconfig/nconf.c
+++ b/scripts/kconfig/nconf.c
@@ -5,7 +5,9 @@
  * Derived from menuconfig.
  *
  */
+#ifndef _GNU_SOURCE
 #define _GNU_SOURCE
+#endif
 #include <string.h>
 #include <stdlib.h>
 
-- 
