From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Thu, 5 Nov 2015 22:02:39 +0100
Subject: [PATCH] {BUG new in 4.4} uninitialized variables

---
 net/tipc/node.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/net/tipc/node.c b/net/tipc/node.c
index 4512e83652b1..052e7373c5b4 100644
--- a/net/tipc/node.c
+++ b/net/tipc/node.c
@@ -689,7 +689,7 @@ static void tipc_node_link_down(struct tipc_node *n, int bearer_id, bool delete)
 {
 	struct tipc_link_entry *le = &n->links[bearer_id];
 	struct tipc_link *l = le->link;
-	struct tipc_media_addr *maddr;
+	struct tipc_media_addr *maddr /* = NULL; TODO: is it useful to silent compiler warnings? */;
 	struct sk_buff_head xmitq;
 	int old_bearer_id = bearer_id;
 
@@ -709,6 +709,7 @@ static void tipc_node_link_down(struct tipc_node *n, int bearer_id, bool delete)
 	} else {
 		/* Defuse pending tipc_node_link_up() */
 		tipc_link_fsm_evt(l, LINK_RESET_EVT);
+		maddr = NULL; /* BUG? */
 	}
 	tipc_node_write_unlock(n);
 	if (delete)
-- 
