From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 22 Oct 2016 11:42:46 +0200
Subject: [PATCH] {CONSTIFY} Constify trace_module_request name parameter

---
 include/trace/events/module.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/trace/events/module.h b/include/trace/events/module.h
index 097485c73c01..ce293e83916d 100644
--- a/include/trace/events/module.h
+++ b/include/trace/events/module.h
@@ -106,7 +106,7 @@ DEFINE_EVENT(module_refcnt, module_put,
 
 TRACE_EVENT(module_request,
 
-	TP_PROTO(char *name, bool wait, unsigned long ip),
+	TP_PROTO(const char *name, bool wait, unsigned long ip),
 
 	TP_ARGS(name, wait, ip),
 
-- 
