From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 8 Oct 2016 15:01:31 +0200
Subject: [PATCH] {CONSTIFY} drivers/pnp/interface: constify format

---
 drivers/pnp/interface.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pnp/interface.c b/drivers/pnp/interface.c
index b084bced2081..2dc1c161b9f1 100644
--- a/drivers/pnp/interface.c
+++ b/drivers/pnp/interface.c
@@ -34,7 +34,7 @@ struct pnp_info_buffer {
 typedef struct pnp_info_buffer pnp_info_buffer_t;
 
 __printf(2, 3)
-static int pnp_printf(pnp_info_buffer_t * buffer, char *fmt, ...)
+static int pnp_printf(pnp_info_buffer_t * buffer, const char *fmt, ...)
 {
 	va_list args;
 	int res;
-- 
