From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 8 Oct 2016 15:01:09 +0200
Subject: [PATCH] {CONSTIFY} drivers/isdn/capi/capiutil: constify bufprint
 format

---
 drivers/isdn/capi/capiutil.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/isdn/capi/capiutil.c b/drivers/isdn/capi/capiutil.c
index 94d7ce148a81..c3bb400dde83 100644
--- a/drivers/isdn/capi/capiutil.c
+++ b/drivers/isdn/capi/capiutil.c
@@ -382,7 +382,7 @@ static char *pnames[] =
 #include <stdarg.h>
 
 /*-------------------------------------------------------*/
-static __printf(2, 3) _cdebbuf *bufprint(_cdebbuf *cdb, char *fmt, ...)
+static __printf(2, 3) _cdebbuf *bufprint(_cdebbuf *cdb, const char *fmt, ...)
 {
 	va_list f;
 	size_t n, r;
-- 
