From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 2 Mar 2015 15:07:49 +0800
Subject: [PATCH] {CONSTIFY} init/main: consitify ramdisk_execute_command

---
 init/main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/main.c b/init/main.c
index ffe16af21a7d..a2e8c2a9bd8f 100644
--- a/init/main.c
+++ b/init/main.c
@@ -140,7 +140,7 @@ static char *static_command_line;
 static char *initcall_command_line;
 
 static char *execute_command;
-static char *ramdisk_execute_command;
+static const char *ramdisk_execute_command;
 
 /*
  * Used to generate warnings if static_key manipulation functions are used
-- 
