From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Tue, 3 Mar 2015 17:13:40 +0800
Subject: [PATCH] {NOT UPSTREAMABLE} drivers/md/bcache/util: fix -Wlogical-op
 warning

---
 drivers/md/bcache/util.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/md/bcache/util.h b/drivers/md/bcache/util.h
index cf2cbc211d83..eedfe339d057 100644
--- a/drivers/md/bcache/util.h
+++ b/drivers/md/bcache/util.h
@@ -140,7 +140,7 @@ do {									\
 									\
 	if (_bytes < KMALLOC_MAX_SIZE)					\
 		(fifo)->data = kmalloc(_bytes, (gfp));			\
-	if ((!(fifo)->data) && ((gfp) & GFP_KERNEL))			\
+	if ((!(fifo)->data) && !!((gfp) & GFP_KERNEL))			\
 		(fifo)->data = vmalloc(_bytes);				\
 	(fifo)->data;							\
 })
-- 
