From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 6 Aug 2016 11:25:26 +0200
Subject: [PATCH] {NOT UPSTREAMABLE} trace_printk: disable -Wformat-security
 locally

---
 samples/trace_printk/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/samples/trace_printk/Makefile b/samples/trace_printk/Makefile
index c0df36167d60..e5fce495c22e 100644
--- a/samples/trace_printk/Makefile
+++ b/samples/trace_printk/Makefile
@@ -5,3 +5,6 @@
 # This module can also be used to test the trace_printk code.
 
 obj-$(CONFIG_SAMPLE_TRACE_PRINTK) += trace-printk.o
+
+# The point of this module is to test things which get reported by clang -Wformat-security ("format string is not a string literal (potentially insecure)")
+CFLAGS_trace-printk.o := -Wno-format-security
-- 
