From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 27 Jun 2015 19:25:48 +0800
Subject: [PATCH] {NOT UPSTREAMABLE} xfs: make STATIC be static

---
 fs/xfs/xfs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/xfs/xfs.h b/fs/xfs/xfs.h
index 80cd0fd86783..0c72dd5b2ff3 100644
--- a/fs/xfs/xfs.h
+++ b/fs/xfs/xfs.h
@@ -19,7 +19,7 @@
 #define __XFS_H__
 
 #ifdef CONFIG_XFS_DEBUG
-#define STATIC
+#define STATIC static
 #define DEBUG 1
 #define XFS_BUF_LOCK_TRACKING 1
 #endif
-- 
