From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 6 Aug 2016 11:17:41 +0200
Subject: [PATCH] {PRINTF} arm/vdso: add printf format attribute to fail()
 function

---
 arch/arm/vdso/vdsomunge.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/vdso/vdsomunge.c b/arch/arm/vdso/vdsomunge.c
index f6455273b2f8..b61faeb06190 100644
--- a/arch/arm/vdso/vdsomunge.c
+++ b/arch/arm/vdso/vdsomunge.c
@@ -93,7 +93,7 @@ static int failed;
 static const char *argv0;
 static const char *outfile;
 
-static void fail(const char *fmt, ...)
+static __attribute__((format(printf, 1, 2))) void fail(const char *fmt, ...)
 {
 	va_list ap;
 
-- 
