From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Tue, 19 Jun 2018 13:16:10 +0200
Subject: [PATCH] {PRINTF} bus/ti-sysc: silence -Wformat-security

---
 drivers/bus/ti-sysc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bus/ti-sysc.c b/drivers/bus/ti-sysc.c
index 2db474ab4c6b..1dc246b078bd 100644
--- a/drivers/bus/ti-sysc.c
+++ b/drivers/bus/ti-sysc.c
@@ -1791,7 +1791,7 @@ static int sysc_child_add_named_clock(struct sysc *ddata,
 	if (IS_ERR(clk))
 		return -ENODEV;
 
-	l = clkdev_create(clk, name, dev_name(child));
+	l = clkdev_create(clk, name, "%s", dev_name(child));
 	if (!l)
 		error = -ENOMEM;
 
-- 
