From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 2 Feb 2020 12:42:02 +0100
Subject: [PATCH] {PRINTF} drivers/dma-buf: use %s to silence -Wformat-security

---
 drivers/dma-buf/dma-heap.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma-buf/dma-heap.c b/drivers/dma-buf/dma-heap.c
index 84ae708fafe7..f0ce4a79e292 100644
--- a/drivers/dma-buf/dma-heap.c
+++ b/drivers/dma-buf/dma-heap.c
@@ -265,7 +265,7 @@ struct dma_heap *dma_heap_add(const struct dma_heap_export_info *exp_info)
 				NULL,
 				heap->heap_devt,
 				NULL,
-				heap->name);
+				"%s", heap->name);
 	if (IS_ERR(dev_ret)) {
 		pr_err("dma_heap: Unable to create device\n");
 		err_ret = ERR_CAST(dev_ret);
-- 
