From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 22 Mar 2019 20:40:32 +0100
Subject: [PATCH] {PRINTF} drivers/remoteproc: silence -Wformat-security

---
 drivers/remoteproc/st_remoteproc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/remoteproc/st_remoteproc.c b/drivers/remoteproc/st_remoteproc.c
index e3ce01d98b4c..54df03931b58 100644
--- a/drivers/remoteproc/st_remoteproc.c
+++ b/drivers/remoteproc/st_remoteproc.c
@@ -142,13 +142,13 @@ static int st_rproc_parse_fw(struct rproc *rproc, const struct firmware *fw)
 						   rmem->size, rmem->base,
 						   st_rproc_mem_alloc,
 						   st_rproc_mem_release,
-						   it.node->name);
+						   "%s", it.node->name);
 		} else {
 			/* Register reserved memory for vdev buffer allocation */
 			mem = rproc_of_resm_mem_entry_init(dev, index,
 							   rmem->size,
 							   rmem->base,
-							   it.node->name);
+							   "%s", it.node->name);
 		}
 
 		if (!mem) {
-- 
