From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 12 May 2019 13:53:50 +0200
Subject: [PATCH] {PRINTF} drivers/usb/typec/tcpm: add __printf attribute

---
 drivers/usb/typec/tcpm/fusb302.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/typec/tcpm/fusb302.c b/drivers/usb/typec/tcpm/fusb302.c
index b28facece43c..14b32fa8bc11 100644
--- a/drivers/usb/typec/tcpm/fusb302.c
+++ b/drivers/usb/typec/tcpm/fusb302.c
@@ -178,7 +178,7 @@ static void _fusb302_log(struct fusb302_chip *chip, const char *fmt,
 	mutex_unlock(&chip->logbuffer_lock);
 }
 
-static void fusb302_log(struct fusb302_chip *chip, const char *fmt, ...)
+static __printf(2, 3) void fusb302_log(struct fusb302_chip *chip, const char *fmt, ...)
 {
 	va_list args;
 
-- 
