From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 15 May 2021 10:31:50 +0200
Subject: [PATCH] {PRINTF} drivers/iommu/sprd-iommu: silence -Wformat-security
 with "%s"

---
 drivers/iommu/sprd-iommu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/iommu/sprd-iommu.c b/drivers/iommu/sprd-iommu.c
index fadd2c907222..ee2e8be2dc17 100644
--- a/drivers/iommu/sprd-iommu.c
+++ b/drivers/iommu/sprd-iommu.c
@@ -488,7 +488,7 @@ static int sprd_iommu_probe(struct platform_device *pdev)
 		goto free_page;
 	}
 
-	ret = iommu_device_sysfs_add(&sdev->iommu, dev, NULL, dev_name(dev));
+	ret = iommu_device_sysfs_add(&sdev->iommu, dev, NULL, "%s", dev_name(dev));
 	if (ret)
 		goto put_group;
 
-- 
