From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 16 Jul 2023 09:21:55 +0200
Subject: [PATCH] {PRINTF} fs/nfs/nfs42xattr: silence -Wformat-security with
 "%s"

---
 fs/nfs/nfs42xattr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/nfs/nfs42xattr.c b/fs/nfs/nfs42xattr.c
index 911f634ba3da..96482a4f247b 100644
--- a/fs/nfs/nfs42xattr.c
+++ b/fs/nfs/nfs42xattr.c
@@ -996,7 +996,7 @@ static int nfs4_xattr_shrinker_init(struct shrinker *shrinker,
 {
 	int ret = 0;
 
-	ret = register_shrinker(shrinker, name);
+	ret = register_shrinker(shrinker, "%s", name);
 	if (ret)
 		return ret;
 
-- 
