From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 16 Dec 2016 09:50:39 +0100
Subject: [PATCH] {PRINTF} i915: silent -Wformat-security

---
 drivers/gpu/drm/i915/intel_hangcheck.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_hangcheck.c b/drivers/gpu/drm/i915/intel_hangcheck.c
index 53df5b11bff4..a0740cdd9f96 100644
--- a/drivers/gpu/drm/i915/intel_hangcheck.c
+++ b/drivers/gpu/drm/i915/intel_hangcheck.c
@@ -430,7 +430,7 @@ static void i915_hangcheck_elapsed(struct work_struct *work)
 					 "%s, ", engine->name);
 		msg[len-2] = '\0';
 
-		return i915_handle_error(dev_priv, hung, msg);
+		return i915_handle_error(dev_priv, hung, "%s", msg);
 	}
 
 	/* Reset timer in case GPU hangs without another request being added */
-- 
