From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 18 Oct 2020 11:11:10 +0200
Subject: [PATCH] {PRINTF} kernel/bpf/btf: remove 0x before %p

---
 kernel/bpf/btf.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/bpf/btf.c b/kernel/bpf/btf.c
index 91d773f5e44a..35b1a8883090 100644
--- a/kernel/bpf/btf.c
+++ b/kernel/bpf/btf.c
@@ -2723,7 +2723,7 @@ static void btf_ptr_show(const struct btf *btf, const struct btf_type *t,
 	if (show->flags & BTF_SHOW_PTR_RAW)
 		btf_show_type_value(show, "0x%px", *(void **)safe_data);
 	else
-		btf_show_type_value(show, "0x%p", *(void **)safe_data);
+		btf_show_type_value(show, "0x %p", *(void **)safe_data);
 	btf_show_end_type(show);
 }
 
-- 
