From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 29 Nov 2019 23:33:02 +0100
Subject: [PATCH] {PRINTF} kunit/string-stream: add printf attribute

---
 lib/kunit/string-stream.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/kunit/string-stream.h b/lib/kunit/string-stream.h
index fe98a00b75a9..1f9b689fa5f4 100644
--- a/lib/kunit/string-stream.h
+++ b/lib/kunit/string-stream.h
@@ -35,7 +35,7 @@ struct string_stream *alloc_string_stream(struct kunit *test, gfp_t gfp);
 int __printf(2, 3) string_stream_add(struct string_stream *stream,
 				     const char *fmt, ...);
 
-int string_stream_vadd(struct string_stream *stream,
+int __printf(2, 0) string_stream_vadd(struct string_stream *stream,
 		       const char *fmt,
 		       va_list args);
 
-- 
