From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Fri, 2 Jul 2021 20:54:15 +0200
Subject: [PATCH] {PRINTF} lib/test_scanf: silence -Wformat-security with %s

---
 lib/test_scanf.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/test_scanf.c b/lib/test_scanf.c
index b620cf7de503..759c75882388 100644
--- a/lib/test_scanf.c
+++ b/lib/test_scanf.c
@@ -337,8 +337,8 @@ static void __init append_delim(char *str_buf, int *str_buf_pos, int str_buf_len
 				char *fmt_buf, int *fmt_buf_pos, int fmt_buf_len,
 				const char *delim_str)
 {
-	append_fmt(str_buf, str_buf_pos, str_buf_len, delim_str);
-	append_fmt(fmt_buf, fmt_buf_pos, fmt_buf_len, delim_str);
+	append_fmt(str_buf, str_buf_pos, str_buf_len, "%s", delim_str);
+	append_fmt(fmt_buf, fmt_buf_pos, fmt_buf_len, "%s", delim_str);
 }
 
 #define test_array_8(fn, check_data, string, fmt, arr)				\
-- 
