From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 15 Aug 2020 12:30:46 +0200
Subject: [PATCH] {PRINTF} mm/cma: silence -Wformat-security

---
 mm/cma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/cma.c b/mm/cma.c
index 54eee2119822..6232e2d5076c 100644
--- a/mm/cma.c
+++ b/mm/cma.c
@@ -197,7 +197,7 @@ int __init cma_init_reserved_mem(phys_addr_t base, phys_addr_t size,
 	cma = &cma_areas[cma_area_count];
 
 	if (name)
-		snprintf(cma->name, CMA_MAX_NAME, name);
+		snprintf(cma->name, CMA_MAX_NAME, "%s", name);
 	else
 		snprintf(cma->name, CMA_MAX_NAME,  "cma%d\n", cma_area_count);
 
-- 
