From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 18 Aug 2018 12:43:39 +0200
Subject: [PATCH] {PRINTF} net/smc: silence -Wformat-security

---
 net/smc/smc_ism.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/smc/smc_ism.c b/net/smc/smc_ism.c
index a2084ecdb97e..4e2109c356b4 100644
--- a/net/smc/smc_ism.c
+++ b/net/smc/smc_ism.c
@@ -418,7 +418,7 @@ struct smcd_dev *smcd_alloc_dev(struct device *parent, const char *name,
 	smcd->dev.parent = parent;
 	smcd->dev.release = smcd_release;
 	device_initialize(&smcd->dev);
-	dev_set_name(&smcd->dev, name);
+	dev_set_name(&smcd->dev, "%s", name);
 	smcd->ops = ops;
 	if (smc_pnetid_by_dev_port(parent, 0, smcd->pnetid))
 		smc_pnetid_by_table_smcd(smcd);
-- 
