From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 29 Dec 2018 10:12:04 +0100
Subject: [PATCH] {PRINTF} scripts/dtc: add PRINTF attribute for
 xavsprintf_append

---
 scripts/dtc/util.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/dtc/util.h b/scripts/dtc/util.h
index ca5cb52928e3..e70d1c4e601d 100644
--- a/scripts/dtc/util.h
+++ b/scripts/dtc/util.h
@@ -59,7 +59,7 @@ extern char *xstrdup(const char *s);
 
 extern int PRINTF(2, 3) xasprintf(char **strp, const char *fmt, ...);
 extern int PRINTF(2, 3) xasprintf_append(char **strp, const char *fmt, ...);
-extern int xavsprintf_append(char **strp, const char *fmt, va_list ap);
+extern int PRINTF(2, 0) xavsprintf_append(char **strp, const char *fmt, va_list ap);
 extern char *join_path(const char *path, const char *name);
 
 /**
-- 
