From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 20 Jul 2019 11:10:46 +0200
Subject: [PATCH] {PRINTF} scripts/basic/fixdep: add __printf attribute to
 xprintf

---
 scripts/basic/fixdep.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/basic/fixdep.c b/scripts/basic/fixdep.c
index 9ba47b0a47b9..3774b86376d3 100644
--- a/scripts/basic/fixdep.c
+++ b/scripts/basic/fixdep.c
@@ -115,7 +115,7 @@ static void usage(void)
  * files. The return value of printf() and putchar() must be checked to catch
  * any error, e.g. "No space left on device".
  */
-static void xprintf(const char *format, ...)
+static __attribute__((format(printf, 1, 2))) void xprintf(const char *format, ...)
 {
 	va_list ap;
 	int ret;
-- 
