From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 9 Jun 2018 16:04:06 +0200
Subject: [PATCH] {PRINTF} scripts/kconfig: add printf format attribute to
 pperror()

---
 scripts/kconfig/preprocess.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/kconfig/preprocess.c b/scripts/kconfig/preprocess.c
index 65da87fce907..921076cb3ea6 100644
--- a/scripts/kconfig/preprocess.c
+++ b/scripts/kconfig/preprocess.c
@@ -14,7 +14,7 @@
 
 static char *expand_string_with_args(const char *in, int argc, char *argv[]);
 
-static void __attribute__((noreturn)) pperror(const char *format, ...)
+static void __attribute__((noreturn)) __attribute__((format(printf, 1, 2))) pperror(const char *format, ...)
 {
 	va_list ap;
 
-- 
