From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 15 May 2021 10:07:46 +0200
Subject: [PATCH] {PRINTF} sound/core/control_led: silence -Wformat-security
 with "%s"

---
 sound/core/control_led.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/core/control_led.c b/sound/core/control_led.c
index 764058cc345d..a270c3d1ca98 100644
--- a/sound/core/control_led.c
+++ b/sound/core/control_led.c
@@ -757,7 +757,7 @@ static int __init snd_ctl_led_init(void)
 		led->dev.parent = &snd_ctl_led_dev;
 		led->dev.release = snd_ctl_led_release;
 		led->dev.groups = snd_ctl_led_dev_attr_groups;
-		dev_set_name(&led->dev, led->name);
+		dev_set_name(&led->dev, "%s", led->name);
 		if (device_add(&led->dev)) {
 			put_device(&led->dev);
 			for (; group > 0; group--) {
-- 
