From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 19 Aug 2018 12:26:13 +0200
Subject: [PATCH] {PRINTF} staging/gasket: silence -Wformat-security

---
 drivers/staging/gasket/gasket_core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/gasket/gasket_core.c b/drivers/staging/gasket/gasket_core.c
index cd8be80d2076..c3ed52460772 100644
--- a/drivers/staging/gasket/gasket_core.c
+++ b/drivers/staging/gasket/gasket_core.c
@@ -228,7 +228,7 @@ static int gasket_alloc_dev(struct gasket_internal_desc *internal_desc,
 		      gasket_dev->dev_idx);
 	dev_info->device =
 		device_create(internal_desc->class, parent, dev_info->devt,
-			      gasket_dev, dev_info->name);
+			      gasket_dev, "%s", dev_info->name);
 
 	/* cdev has not yet been added; cdev_added is 0 */
 	dev_info->gasket_dev_ptr = gasket_dev;
-- 
