From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sun, 12 Mar 2023 09:05:00 +0100
Subject: [PATCH] {PRINTF} tools/lib/subcmd: add printf format attribute

---
 tools/lib/subcmd/subcmd-util.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tools/lib/subcmd/subcmd-util.h b/tools/lib/subcmd/subcmd-util.h
index dfac76e35ac7..787cfc6ee967 100644
--- a/tools/lib/subcmd/subcmd-util.h
+++ b/tools/lib/subcmd/subcmd-util.h
@@ -7,14 +7,14 @@
 #include <stdio.h>
 #include <linux/compiler.h>
 
-static inline void report(const char *prefix, const char *err, va_list params)
+static __attribute__((format(printf, 2, 0))) inline void report(const char *prefix, const char *err, va_list params)
 {
 	char msg[1024];
 	vsnprintf(msg, sizeof(msg), err, params);
 	fprintf(stderr, " %s%s\n", prefix, msg);
 }
 
-static __noreturn inline void die(const char *err, ...)
+static __noreturn __attribute__((format(printf, 1, 2))) inline void die(const char *err, ...)
 {
 	va_list params;
 
-- 
