From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 21 Sep 2019 20:27:51 +0200
Subject: [PATCH] {MAYBE UPS} drivers/android/binderfs: fix clang's
 -Wmissing-braces

---
 drivers/android/binderfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/android/binderfs.c b/drivers/android/binderfs.c
index e2580e5316a2..bde10d5b674d 100644
--- a/drivers/android/binderfs.c
+++ b/drivers/android/binderfs.c
@@ -654,7 +654,7 @@ static int binderfs_fill_super(struct super_block *sb, void *data, int silent)
 	int ret;
 	struct binderfs_info *info;
 	struct inode *inode = NULL;
-	struct binderfs_device device_info = { 0 };
+	struct binderfs_device device_info = { { 0 } };
 	const char *name;
 	size_t len;
 
-- 
