From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 20 Dec 2021 17:35:43 +0100
Subject: [PATCH] {MAYBE UPS} drivers/net/ethernet/ti: mark unused hw-related
 variable

---
 drivers/net/ethernet/ti/tlan.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/net/ethernet/ti/tlan.c b/drivers/net/ethernet/ti/tlan.c
index b6923dfe88fd..2f6d094590a6 100644
--- a/drivers/net/ethernet/ti/tlan.c
+++ b/drivers/net/ethernet/ti/tlan.c
@@ -2117,6 +2117,7 @@ static void tlan_read_and_clear_stats(struct net_device *dev, int record)
 		dev->stats.tx_aborted_errors += tx_under;
 		dev->stats.tx_carrier_errors += loss;
 	}
+	(void) def_tx; /* Mark the variable as unused to please clang's -Wunused-but-set-variable */
 
 }
 
-- 
