From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Mon, 20 Dec 2021 17:14:48 +0100
Subject: [PATCH] {MAYBE UPS} drivers/usb/host/u132-hcd: remove unused variable

---
 drivers/usb/host/u132-hcd.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/host/u132-hcd.c b/drivers/usb/host/u132-hcd.c
index d879d6af5710..ea4f66e66372 100644
--- a/drivers/usb/host/u132-hcd.c
+++ b/drivers/usb/host/u132-hcd.c
@@ -2335,14 +2335,12 @@ static int u132_urb_enqueue(struct usb_hcd *hcd, struct urb *urb,
 			char data[30 * 3 + 4];
 			char *d = data;
 			int m = (sizeof(data) - 1) / 3;
-			int l = 0;
 			data[0] = 0;
 			while (urb_size-- > 0) {
 				if (i > m) {
 				} else if (i++ < m) {
 					int w = sprintf(d, " %02X", *b++);
 					d += w;
-					l += w;
 				} else
 					d += sprintf(d, " ..");
 			}
-- 
