From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 28 Feb 2015 11:07:33 +0800
Subject: [PATCH] {MAYBE UPS} (grsecurity) Add some changes from grsec

---
 net/socket.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/net/socket.c b/net/socket.c
index 9b27c5e4e5ba..db5a57f95390 100644
--- a/net/socket.c
+++ b/net/socket.c
@@ -1455,6 +1455,8 @@ int __sock_create(struct net *net, int family, int type, int protocol,
 		return -EAFNOSUPPORT;
 	if (type < 0 || type >= SOCK_MAX)
 		return -EINVAL;
+	if (protocol < 0)  /* Found in grsecurity patch 2015-02-27 */
+		return -EINVAL;
 
 	/* Compatibility.
 
@@ -2147,7 +2149,7 @@ SYSCALL_DEFINE4(send, int, fd, void __user *, buff, size_t, len,
 int __sys_recvfrom(int fd, void __user *ubuf, size_t size, unsigned int flags,
 		   struct sockaddr __user *addr, int __user *addr_len)
 {
-	struct sockaddr_storage address;
+	struct sockaddr_storage address = {};
 	struct msghdr msg = {
 		/* Save some cycles and don't copy the address if not needed */
 		.msg_name = addr ? (struct sockaddr *)&address : NULL,
-- 
