From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 28 Feb 2015 11:07:33 +0800
Subject: [PATCH] {MAYBE UPS} (grsecurity) Add some changes from grsec

---
 net/socket.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/net/socket.c b/net/socket.c
index 01f3f8f32d6f..ec50d82ee69a 100644
--- a/net/socket.c
+++ b/net/socket.c
@@ -1217,6 +1217,8 @@ int __sock_create(struct net *net, int family, int type, int protocol,
 		return -EAFNOSUPPORT;
 	if (type < 0 || type >= SOCK_MAX)
 		return -EINVAL;
+	if (protocol < 0)  /* Found in grsecurity patch 2015-02-27 */
+		return -EINVAL;
 
 	/* Compatibility.
 
@@ -1821,7 +1823,7 @@ int __sys_recvfrom(int fd, void __user *ubuf, size_t size, unsigned int flags,
 	struct socket *sock;
 	struct iovec iov;
 	struct msghdr msg;
-	struct sockaddr_storage address;
+	struct sockaddr_storage address = { };  /* Found in grsecurity patch 2015-02-27 */
 	int err, err2;
 	int fput_needed;
 
@@ -2252,7 +2254,7 @@ static int ___sys_recvmsg(struct socket *sock, struct user_msghdr __user *msg,
 	ssize_t err;
 
 	/* kernel mode address */
-	struct sockaddr_storage addr;
+	struct sockaddr_storage addr = { };  /* Found in grsecurity patch 2015-02-27 */
 
 	/* user mode address pointers */
 	struct sockaddr __user *uaddr;
-- 
