From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 28 Feb 2015 11:07:33 +0800
Subject: [PATCH] {MAYBE UPS} (grsecurity) Add some changes from grsec

---
 net/socket.c | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/net/socket.c b/net/socket.c
index bf2122691fba..b551f48b75bb 100644
--- a/net/socket.c
+++ b/net/socket.c
@@ -1188,6 +1188,8 @@ int __sock_create(struct net *net, int family, int type, int protocol,
 		return -EAFNOSUPPORT;
 	if (type < 0 || type >= SOCK_MAX)
 		return -EINVAL;
+	if (protocol < 0)  /* Found in grsecurity patch 2015-02-27 */
+		return -EINVAL;
 
 	/* Compatibility.
 
@@ -1764,7 +1766,7 @@ SYSCALL_DEFINE6(recvfrom, int, fd, void __user *, ubuf, size_t, size,
 	struct socket *sock;
 	struct iovec iov;
 	struct msghdr msg;
-	struct sockaddr_storage address;
+	struct sockaddr_storage address = { };  /* Found in grsecurity patch 2015-02-27 */
 	int err, err2;
 	int fput_needed;
 
@@ -2167,7 +2169,7 @@ static int ___sys_recvmsg(struct socket *sock, struct user_msghdr __user *msg,
 	ssize_t err;
 
 	/* kernel mode address */
-	struct sockaddr_storage addr;
+	struct sockaddr_storage addr = { };  /* Found in grsecurity patch 2015-02-27 */
 
 	/* user mode address pointers */
 	struct sockaddr __user *uaddr;
@@ -2815,7 +2817,7 @@ static int ethtool_ioctl(struct net *net, struct compat_ifreq __user *ifr32)
 	ifr = compat_alloc_user_space(buf_size);
 	rxnfc = (void __user *)ifr + ALIGN(sizeof(struct ifreq), 8);
 
-	if (copy_in_user(&ifr->ifr_name, &ifr32->ifr_name, IFNAMSIZ))
+	if (copy_in_user(ifr->ifr_name, ifr32->ifr_name, IFNAMSIZ))  /* Found in grsecurity patch 2015-02-27 */
 		return -EFAULT;
 
 	if (put_user(convert_in ? rxnfc : compat_ptr(data),
-- 
