From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 15 Aug 2020 20:35:33 +0200
Subject: [PATCH] {MAYBE UPS} Remove self-assignments

---
 drivers/crypto/sa2ul.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/crypto/sa2ul.c b/drivers/crypto/sa2ul.c
index 5bc099052bd2..0cc8eb60adaa 100644
--- a/drivers/crypto/sa2ul.c
+++ b/drivers/crypto/sa2ul.c
@@ -1149,7 +1149,7 @@ static int sa_run(struct sa_req *req)
 				       &split_size, &dst, &dst_nents,
 				       gfp_flags);
 			if (ret) {
-				dst_nents = dst_nents;
+				//dst_nents = dst_nents;
 				dst = req->dst;
 			} else {
 				rxd->split_dst_sg = dst;
-- 
